src/main/java/org/aksw/simba/autoindex/es/repository/EntityRepository.java

```diff
@@ -136,16 +136,13 @@ public Response search(SearchRequest searchRequest) {
 		
 		String strType = getType(type);
 		if(query.contains("*") || query.contains("?") ) {
-			System.out.println("Wild card Query");
-			nativeSearchQueryBuilder.withQuery(QueryBuilders.queryStringQuery(query)).withPageable(new PageRequest(0, 100));
+			nativeSearchQueryBuilder.withQuery(QueryBuilders.queryStringQuery(query)).withPageable(new PageRequest(0, 1000));
 		}
 		else if(Type.URI.equals(type)) {
-			System.out.println("URI Search");
-			nativeSearchQueryBuilder.withQuery(QueryBuilders.matchQuery(strType , query)).withPageable(new PageRequest(0, 100));
+			nativeSearchQueryBuilder.withQuery(QueryBuilders.matchQuery(strType , query)).withPageable(new PageRequest(0, 1000));
 		}
 		else {
-			//TODO: Add Fuzzy Search (0,1,2)
-			nativeSearchQueryBuilder.withQuery(QueryBuilders.matchQuery(strType , query)).withPageable(new PageRequest(0, 100));
+			nativeSearchQueryBuilder.withQuery(QueryBuilders.matchQuery(strType , query).fuzziness(1).prefixLength(0).maxExpansions(2)).withPageable(new PageRequest(0, 1000));

```

Comment:
@RicardoUsbeck Are these the correct parameters for searching for labels(& maybe URL's)? Edit distance of 1,  Prefix Length of 1 and Expansions allowed = 2. Do we also need Fuzzy Transpositions to be enabled?